src/components/Header.js

```diff
@@ -0,0 +1,11 @@
+import React from 'react'
```

Comment:
Consider adding this header directly into App.js. Since there are no props or state, this doesn't really require it's own component.